controller/space_areas_blackbox_test.go

```diff
@@ -132,7 +132,7 @@ func (rest *TestSpaceAreaREST) TestListAreasOKUsingExpiredIfModifiedSinceHeader(
 	parentArea, createdAreaUuids, _ := rest.setupAreas()
 	// when
 	ifModifiedSince := app.ToHTTPTime(parentArea.UpdatedAt.Add(-1 * time.Hour))
-	res, areaList := test.ListSpaceAreasOK(rest.T(), rest.svcSpaceAreas.Context, rest.svcSpaceAreas, rest.ctrlSpaceAreas, parentArea.SpaceID.String(), &ifModifiedSince, nil)
+	res, areaList := test.ListSpaceAreasOK(rest.T(), rest.svcSpaceAreas.Context, rest.svcSpaceAreas, rest.ctrlSpaceAreas, parentArea.SpaceID, &ifModifiedSince, nil)
```

Comment:
thanks ;) there's also a lot of removal of `string` -> `uuid.UUID` conversion, as you'll see.